hrvisa/tests/phpunit/CRM/AllTests.php

```diff
@@ -1,68 +0,0 @@
-<?php
```

Comment:
Actually, I noticed there are a lot of AllTests.php files left to be removed:
./hrabsence/tests/phpunit/AllTests.php
./hrabsence/tests/phpunit/api/v3/AllTests.php
./hrbank/tests/phpunit/WebTest/AllTests.php
./hrcareer/tests/phpunit/WebTest/AllTests.php
./hrcase/tests/phpunit/WebTest/AllTests.php
./hremerg/tests/phpunit/WebTest/AllTests.php
./hrident/tests/phpunit/WebTest/AllTests.php
./hrjob/tests/phpunit/AllTests.php
./hrjob/tests/phpunit/api/v3/AllTests.php
./hrjob/tests/phpunit/CRM/AllTests.php
./hrjob/tests/phpunit/WebTest/AllTests.php
./hrmed/tests/phpunit/WebTest/AllTests.php
./hrqual/tests/phpunit/WebTest/AllTests.php
./hrrecruitment/tests/phpunit/WebTest/AllTests.php
./hrstaffdir/tests/phpunit/WebTest/AllTests.php
./hrvisa/tests/AllTests.php
./hrvisa/tests/phpunit/WebTest/AllTests.php
./uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/AllTests.php
./uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/WebTest/AllTests.php

(Yeah, I forgot to remove the ones in L&A :( )